### PR TITLE
Refactoring grounding a bit

### PR DESCRIPTION
**Pull request by feiranwang**

### CODE REVIEW

File: src/main/scala/org/deepdive/inference/datastore/SQLInferenceDataStore.scala [hunk 1]
```diff
@@ -640,10 +569,42 @@ trait SQLInferenceDataStore extends InferenceDataStore with Logging {
...
+      CREATE TABLE ${VariablesObservationTable}(variable_id bigint primary key);
+      """)
+    calibrationSettings.observationQuery match {
+      case Some(query) => execute(query)
+      case None =>
+    }
+  }
+
+  // check whether greenplum is used
+  def isUsingGreenplum() : Boolean = {
```
Comment (zifeishan) [inline src/main/scala/org/deepdive/inference/datastore/SQLInferenceDataStore.scala]: Is it better to move this to Helper.scala? @feiranwang

**Comment by zifeishan** [comment]
Looks good to me other than the comments. Have you tested on a GP machine? @feiranwang

File: src/main/scala/org/deepdive/inference/datastore/SQLInferenceDataStore.scala (hunk 1 above)
Comment (feiranwang) [inline src/main/scala/org/deepdive/inference/datastore/SQLInferenceDataStore.scala]: I'll check that @zifeishan

**Comment by feiranwang** [comment]
Not yet... will test it. Thanks! @zifeishan

**Comment by chrismre** [comment]
Hi Guys, I wrote a code review that I shared with you via Google docs... this file needs a refactor. Let's get that done!

**Pull request closed without merging**